ts/deck-options/SaveButton.svelte

```diff
@@ -29,6 +29,12 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         }
     }
 
+    /// Treat text like HTML, merging multiple spaces and converting
+    /// newlines to spaces.
+    export function collapseWhitespace(s: string): string {
+        return s.replace(/\s+/g, " ");
```

Comment:
The reason this was in i18n is because it's useful when you have a multi-line ftl string that you want to show in a non-HTML context. In HTML the whitespace is collapsed, but in a text context, you end up with unwanted newlines. There's a with_collapsed_whitespace() in the Python code as well.